src/main/java/com/browserup/harreader/model/HarLog.java

```diff
@@ -123,4 +129,41 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(version, creator, browser, pages, entries, comment);
     }
+
+    /**
+     * Search the entire log for the most recent entry whose request URL matches the given <code>url</code>.
```

Comment:
Could we add a few realistic regexp examples to our doc?  I've seen a lot of cases where messed up regexps make users think things "aren't working" so examples might be helpful. Classes of example regexps that might be handy to remind users of: 1-a standard one that matches http/https like you have in your test.  2-One that allows for any parameters, 3-One that wildcards a UUID in the middle of a regexp, 4-One that wildcards host/port.